src/Events/AbsctactBoundFilterEvent.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * This file is part of OpenEMR.
+ *
+ * @link https://github.com/openemr/openemr/tree/master
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+namespace OpenEMR\Events;
+
+use Symfony\Component\EventDispatcher\Event;
+
+/**
+ * Event object template for creating bound filter events
+ *
+ * @package OpenEMR\Events
+ * @author Ken Chapple <ken@mi-squared.com>
+ * @copyright Copyright (c) 2019 Ken Chapple <ken@mi-squared.com>
+ */
+abstract class AbsctactBoundFilterEvent extends Event
```

Comment:
AbsctactBoundFilterEvent mispelled. this can be fixed after in codebase since it is consistently mispelled :)